tests/molecule/resources/converge.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Provision infrastructure
+  hosts: all
+  become: true
+  gather_facts: true
+  roles:
+    - role: mirsg.infrastructure.provision
+      tags: provision
+    - role: mirsg.infrastructure.install_python
+      tags: python
```

Comment:
we will need to add the other roles here, and probably target specific groups for some of them